variables.tf

```diff
@@ -59,37 +59,49 @@ variable "dead_letter_queue_destination" {
   description = "Send failed events/function executions to a dead letter queue arn sns or sqs"
 }
 
-variable "subscribed_log_group_names" {
```

Comment:
Removing this would trigger a backward incompatible change. Is there any way of supporting both `subscribed_log_group_names` and `subscribed_log_group_matches`, but making them mutually exclusive?